src/utils/css.js

```diff
@@ -2,6 +2,6 @@
 Backdraft.Utils.toCSSClass = (function() {
   var cssClass = /[^a-zA-Z_0-9\-]/g;
   return function(input) {
-    return input.replace(cssClass, "-");
+    return "column-"+input.replace(cssClass, "-");
```

Comment:
Something missed from our discussions that would be great to throw in.. `column-` appears now 3 times in the repo, and isn't dry (or prone to updating without causing bugs).

How about:

```
Backdraft.Utils.CSS_CLASS_PREFIX = "column-";
```